### PR TITLE
v0.3.1 Token types fix

### PR DESCRIPTION
**Pull request by Tristyn**
This is a fix for an undefined error introduced in 0.3.0

Commit history is a bit messy because I branched off token-types instead of master, but it'll be fine

### CODE REVIEW

File: examples/user_position.ts [hunk 1]
```diff
@@ -1,72 +1,72 @@
...
-}
-
-async function getBitcoinPosition() {
-  // This users positions will be fetched
-  const userAddress = new PublicKey("6XEn2q37nqsYQB5R79nueGi6n3uhgjiDwxoJeAVzWvaS");
-  //transaction commitment options
-  const options = Provider.defaultOptions();
-  const connection = new Connection("https://api.devnet.solana.com", options);
-  // A wallet is not required in this example
-  const wallet = undefined as any as typeof Wallet;
```
Comment (Tristyn) [inline examples/user_position.ts:26]: This was the only line I changed. I think the file has line endings changed

File: src/common/associatedToken.ts [hunk 2]
```diff
@@ -73,8 +73,7 @@ export class AssociatedToken {
       if (!account) {
         return undefined
       }
-      //TODO - FIXME cast it as any for now
-      const info = parseTokenAccount(account.data as any, addresses[i])
+      const info = parseTokenAccount(account, addresses[i])
```
Comment (Tristyn) [inline src/common/associatedToken.ts:76]: `as any` caused the compiler to not check types. In the future we're moving to getting autogenerated types directly from the IDL which will cut down on as any shenanigans. `as any`, `thisCouldBeNull!` and other syntax to turn off type checks are code smells

Comment (jetjake) [reply src/common/associatedToken.ts:76]: nice, removing the shenanigans

**Review by jetjake — APPROVED** [review]
Tested it and didn't get the issue that was seen earlier today.  Nice Tristyn

**Pull request merged**